src/System.Private.ServiceModel/src/System/ServiceModel/ClientBase.cs

```diff
@@ -17,124 +15,30 @@ public abstract class ClientBase<TChannel> : ICommunicationObject, IDisposable
         where TChannel : class
     {
         private TChannel _channel;
-        private ChannelFactoryRef<TChannel> _channelFactoryRef;
-        private EndpointTrait<TChannel> _endpointTrait;
```

Comment:
I don't believe EndpointTrait and any of the derived classes are used any more. Can you check if it's safe to remove them?